back/src/main/java/com/ubikloadpack/jmeter/ulp/observability/util/ErrorTypeInfo.java

```diff
@@ -0,0 +1,64 @@
+package com.ubikloadpack.jmeter.ulp.observability.util;
+
+/**
+ * ErrorTypeInfo holds the type of an error and it's occurrences.
+ * It defines the method like {@link #computeErrorTypeFrequency} to get the frequency 
+ * of that error type. Also you can get the rate error by calling 
+ * the {@link #computeErrorRate} method.
+ */
+public class ErrorTypeInfo implements Comparable<ErrorTypeInfo> {
+	
+	private String errorType;
+	private Long occurence;

```

Comment:
Why make it a Wrapper instead of long ?

Reply:
I didn't pay attention to that. In fact, it may just be long.